### PR TITLE
Small UI/UX improvements

### PR DESCRIPTION
**Pull request by chuckbergeron**

### CODE REVIEW

File: lib/components/LandingPage.jsx [hunk 1]
```diff
@@ -7,15 +7,15 @@ import {
...
-import classnames from 'classnames'
 import React from 'react'
 import { Trans, useTranslation } from 'react-i18next'
 
-import ChatBubble from 'assets/images/LandingPage/chat-bubble.png'
-import DollarBill from 'assets/images/LandingPage/dollar-bill.png'
-import VoteIcon from 'assets/images/LandingPage/vote-icon.png'
 import { DISCORD_INVITE_URL, POOLTOGETHER_GOV_FORUM_URL } from 'lib/constants'
 
+import ChatBubble from 'assets/images/chat-bubble@2x.png'
```
Comment (ddesrosier) [inline lib/components/LandingPage.jsx:15]: We should start using just flattened svg's with fills. They scale and look significantly better & we could customize them (set a gradient fill, purple, teal, hover effects etc). Pngs are always so pixelated.

The gradients on these look funny - the dots in the chat bubble & dollar bill don't fit the rest of the image.

**Review by ddesrosier — COMMENTED** [review]
I thought we wanted to get rid of this state?

Screen Shot 2021-09-01 at 11 52 27 AM

**Comment by chuckbergeron** [comment]
> I thought we wanted to get rid of this state?
> 
> Screen Shot 2021-09-01 at 11 52 27 AM

I think this works for now, but what were you thinking by "get rid of it"? Are you thinking it would be better to show a different state if they hold POOL but are undelegated?

**Comment by ddesrosier** [comment]
> > I thought we wanted to get rid of this state?
> > Screen Shot 2021-09-01 at 11 52 27 AM
> 
> I think this works for now, but what were you thinking by "get rid of it"? Are you thinking it would be better to show a different state if they hold POOL but are undelegated?

Ohh I didn't realize the top sentence changed to clarify the big vote count. Nvm 😅

**Pull request merged**